crates/router/src/core/payment_methods/cards.rs

```diff
@@ -17,7 +17,7 @@ use common_utils::{
 };
 use error_stack::{report, IntoReport, ResultExt};
 use router_env::{instrument, tracing};
-use storage_models::payment_method;
+use storage_models::{enums::CaptureMethod, payment_method};
```

Comment:
avoid specific imports, instead import `enums` and in the later part of the code use `enums::CaptureMethod` instead